### ISSUE TITLE
Need to be able to delete an order from the website 

### ISSUE THREAD

**Issue by Xi23000** [Xi23000/qb1#12]
Need to be able to delete orders

**Comment by consuspk** [comment]
In next Phase

**Comment by consuspk** [comment]
Do you want to delete orders from QB also if already synced with QB .?

**Comment by Xi23000** [comment]
Yes, it should also delete from QB, unless it has been closed or Invoiced



![image](https://cloud.githubusercontent.com/assets/25314838/22380549/daf30038-e48b-11e6-8483-2eb8d422e7e2.png)
